tests/baselines/reference/commentsOnRequireStatement.js

```diff
@@ -32,7 +32,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 // blah 
 // blah 
 var _0_1 = require("./0");
-exports.subject = _0_1.subject;
+Object.defineProperty(exports, "subject", { enumerable: true, get: function () { return _0_1.subject; } });

```

Comment:
Any reason not to just use the `__createBinding` helper for these cases?

Reply:
We use `__createBinding` when targeting es3; we use this when targeting `es5` (since the assumption is when targeting es5, we don't need the conditional behavior of `__createBinding`).